### DISCUSSION TITLE
Vereinfachtes Feedback

### DISCUSSION THREAD

**Discussion by OpenGreenStreet** [streetcomplete/StreetComplete#4352] [Ideas]
Bis vor ein paar Monaten konnte man sehr einfach Feedback per E-Mail geben. Das war super. Als das schließlich abgeschafft wurde, konnte ich gut nachvollziehen warum: auf Dauer nicht von einer einzelnen Person so leistbar. Die heutige Form des Feedback ist (zumindest aus meiner Sicht) für den Feedbackgebenden sehr umständlich. GitHub bietet natürlich super Funktionen um eine strukturierte Abarbeitung zu ermöglichen. Ich sehe aus Sicht des Feedbackgebenden aber zwei entscheidende Nachteile:

* Ich muss mich anmelden.

* Am Smartphone ist die Erstellung eines Issues sehr umständlich

Diese beiden Punkte haben mich schon mehrmals davon abgehalten, Hinweise einzureichen.

Wäre es nicht möglich, etwas vergleichbares wie für die Wikimedia Commons App zu schaffen:

https://commons.wikimedia.org/wiki/Commons:Mobile_app/Feedback

* aus der App heraus (ohne zusätzliche Anmeldung, man ist ja bereits bei SC angemeldet) eine Hinweis auf einer Website erstellen. 

Wenn ein solcher Hinweis Relevantes enthält, legt ein erfahrener Mapper daraus ein Issue in GitHub an. Wenn nicht bleibt der Hinweis halt so stehen.

**Comment by legofahrrad** [comment]
GitHub ist eben, wie du schon sagst, einfach das bessere Tool anstatt von E-Mails. Hier können Issues einfach kommentiert werden, sie können in Kategorien durch Label sortiert werden und auch das Markieren als "offen"/"erledigt" ist sehr praktisch.
All das ist bei E-Mails nicht möglich.
Deine Nachteile kann ich auch verstehen, aber das Anmelden ist eine einmalige Sache, danach kann man das Passwort vom Browser speichern lassen und ist somit immer eingeloggt, wenn man GitHub besucht. Für das Problem am Smartphone habe auch ich noch keine Lösung, ich finde es auch umständlicher als am Computer. Das ist einfach so ;)
Alles in allem finde ich also das System hier bei GitHub ganz gut, auch durch das "neue" [Dishussionsforum](https://github.com/streetcomplete/StreetComplete/discussions) lassen sich Ideen gut diskutieren, bevor man ein Issue dafür erstellt.

**Comment by OpenGreenStreet** [comment]
@legofahrrad Danke für Deine Rückmeldung: wie geschrieben, kann ich die Vorteile von GitHub gut nachvollziehen. Für Quick and Dirty ist es aber leider eher nicht, zumal ich gerne beim Schließen des Browser alle Daten löschen lasse und damit auch die Anmeldung weg ist . . .
Wie auch geschrieben fände ich einen „Zwischenschritt“ wie hier: https://commons.wikimedia.org/wiki/Commons:Mobile_app/Feedback super

**Comment by OpenGreenStreet** [comment]
So könnte auch gleich die „Spreu vom Weizen getrennt“ werden :-)

**Comment by mnalis** [comment]
> As also written I would find an "intermediate step" like here: https://commons.wikimedia.org/wiki/Commons:Mobile_app/Feedback super

That would require someone to go over those intermediate steps, and create appropriate SC [issues](https://github.com/streetcomplete/StreetComplete/issues) from those. Also, it would (by default) lose ability to ask original reporter extra questions when issue is commented on (which are quite often needed) which GitHub handles automatically (and also allows users to customize to their needs/wants). 

And, such an intermediate form would need to be made more complex (in fact, it should be made as complex as GitHub issue is, in order to collect all that data that GitHub collects!), **_or_** it would need even more triaging work from the developers to get needed data. So it would not magically get easier for the user; it is just that what is currently with github _"just one, but more complex step"_ would be split to _"**several** less complex steps"_, in the end requiring **more** additional work from both developers and users reporting the problem.

**Comment by OpenGreenStreet** [comment]
Danke, das kann ich wie gesagt alles sehr gut nachvollziehen.

Ich wollte nur darauf hinweisen, dass aus meiner Sicht durch das veränderte Feedback die Hemmschwelle zur Rückmeldung von Problemen und Wünschen deutlich erhöht wurde.

**Comment by OpenGreenStreet** [comment]
Ich habe gerade bei Vespucci ein Feedback gegeben. Dort funktioniert es auch über GitHub ohne(!) Anmeldung (inklusive Daten der App). 

Könntet Ihr das vielleicht ähnlich umsetzen oder die Authentifizierung von StreetComplete an GitHub weiterreichen?

**Reply by mnalis** [reply]
When I go to `Provide feedback` option in Vespucci, it asks me to login to GitHub. It probably doesn't ask you as you have already logged in while reporting an issue for StreetComplete! So, probably StreetComplete also doesn't ask you to login to GitHub _anymore_.

**Comment by westnordost** [comment]
Ne, aber was man zukünftig in Erwägung ziehen könnte, ist, das Github Discussions Forum dicht zu machen und stattdessen ein Unterforum auf community.openstreetmap.org zu haben.
Vorteile:
- StreetComplete Nutzer wird gleich der Ort gezeigt, an dem die OSM Community im ganzen sonst noch so erreichbar ist
- Erstellen eines Accounts nicht notwendig da man sich einfach mit dem OpenStreetMap Account einloggt
- Unkompliziererer Support dafür dass man in der Sprache seiner Wahl postet (Übersetzen per Button-Click)

**Comment by OpenGreenStreet** [comment]
Hi Tobias,

was insgesamt die optimale Lösung ist kann ich nicht einschätzen. Im Vergleich zum direkten E-Mail-Kontakt zu Dir ist die gegenwärtige Lösung aus meiner Sicht ein deutlicher Rückschritt/deutlich unkomfortabler.

Die Lösung der Commons-App oder der Vespucci-App gefällt mir deutlich besser. (Aber was geht und was nicht geht kann ich nicht beurteilen.)

Viele Grüße 

Armin

**Reply by mnalis** [reply]
I get it that reporting problems become more complicated for users than just using e-mail...

However, note that the likely result of **_ever growing userbase_** e-mailing one man would much **more likely** result in that man becoming **burnt-out** and abandoning the project altogether, rather than him continuing to (manually and single-handedly) process _ever growing_ amount of incoming e-mails while lifeforce is being sucked out of him.

Changing the bug reporting way to GitHub, while admittedly making it _slightly_ more inconvenient for the users, allows other people to share a part of that load from the project owner, allowing him to continue.

That is the curse of popularity. 
- If the project had just few users, we could all meet and discuss each issue over beer; that would be most pleasant I imagine! :smiley: 
- With several hundreds of users, beer in person is no longer an option :cry:, but e-mail works just fine. 
- With tens of thousands of users, e-mail is no longer an option, but GitHub-like solution works OK
- If we get to millions of users, it is likely that GitHub won't be enough, and you'd have to have specialized solution for triaging and get ready for each issue to take months/years getting through several layers of tech support until the issue gets solved (or more likely, deemed not important enough and simply ignored). So, it is time to rejoice we have not _yet_ reached this level of popularity :smiling_face_with_tear:

**Reply by Helium314** [reply]
Plus with e-mail you don't see what was already discussed with others, greatly increasing the chance for duplicate issues / requests.
